metadata.json

```diff
@@ -12,15 +12,18 @@
         ],
         "Category": "Application Launchers",
         "Description": "An improved fork of Kicker Application Menu which is a launcher based on cascading popup menus",
+        "Category[nl]": "Beter applicatiemenu",
```

Comment:
Category should be a generic name i.e `Application Launchers` and not the specific name i.e `Better Application Launcher`